program-manager/src/program_config.rs

```diff
@@ -212,8 +232,8 @@ impl ProgramConfig {
 
                 // Add processor bridge account info by domain
                 self.authorization_data
-                    .set_processor_bridge_addr(domain.clone(), processor_bridge_account_addr);
-            };
+                    .set_processor_bridge_addr(processor_bridge_account_addr);
+            }
```

Comment:
do you not need to know the domain of that processor bridge account? How is this change not breaking anything?